pkg/alertmanager/operator.go

```diff
@@ -1086,6 +1093,11 @@ func (c *Operator) selectAlertmanagerConfigs(ctx context.Context, am *monitoring
 				"namespace", am.Namespace,
 				"alertmanager", am.Name,
 			)
+			if c.canEmitEvents {
+				c.eventRecorder.Eventf(amc, v1.EventTypeWarning, "InvalidConfiguration", "AlertmanagerConfig %s was rejected due to invalid configuration: %v", amc.GetName(), err)
+			} else {
+				level.Debug(c.logger).Log("msg", "skipping event emission for InvalidConfiguration due to missing RBAC permissions")
```

Comment:
I would log it once during operator startup

Reply:
I'm always in doubt if debug logs should be abused or not 😅 

I guess once is enough to capture the problem